### DISCUSSION TITLE
middleware/redis/user-guide/create/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1927] [Announcements]
# middleware/redis/user-guide/create/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/redis/user-guide/create/

**Comment by liyuerich** [comment]
概览，实例监控，备份管理 能否也介绍一下？

**Comment by liyuerich** [comment]
部署类型： 主备模式 现在是 哨兵模式， 图片中还是主备模式

**Reply by ModetaNiu** [reply]
已修复 见 PR #4850

**Comment by liyuerich** [comment]
可以对哨兵模式部署架构简单介绍一下，还是和单机，集群模式不太一样

**Reply by windsonsea** [reply]
Add some description in PR #5324
